### PR TITLE
DEX - updating 686 backup submission to support multiple pdfs

### PR DESCRIPTION
**Pull request by tblackwe**
## Summary

When a user submits a 686 + 674, we want to bundle the 2 claims together and submit to CMS as 1 submission.
This PR resolves some issues found during acceptance testing

## Related issue(s)
https://app.zenhub.com/workspaces/benefits-dependents-experience-6459139e7d64bf0015f6af13/issues/gh/department-of-veterans-affairs/va.gov-team/64538

## Testing done

- Spec tests updated
- Claim submitted in review branch, logs reviewed to ensure proper submission

## What areas of the site does it impact?
686/674 Backup Submissions

## Acceptance criteria

- [X]  I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [X]  No error nor warning in the console.
- [X]  Events are being sent to the appropriate logging solution
- [ ]  Documentation has been updated (link to documentation)
- [X]  No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [X]  Feature/bug has a monitor built into Datadog or Grafana (if applicable)
- [ ]  If app impacted requires authentication, did you login to a local build and verify all authenticated routes work as expected
- [ ]  I added a screenshot of the developed feature

## Requested Feedback

(OPTIONAL)_What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

File: app/sidekiq/central_mail/submit_central_form686c_job.rb [hunk 1]
```diff
@@ -14,15 +14,15 @@ class SubmitCentralForm686cJob
...
+    attr_reader :claim, :form_path, :attachment_paths
 
     class CentralMailResponseError < StandardError; end
 
     def extract_uuid_from_central_mail_message(data)
       data.body[/(?<=\[).*?(?=\])/].split(': ').last if data.body.present?
     end
 
-    def perform(saved_claim_id, encrypted_vet_info, encrypted_user_struct)
+    def perform(saved_claim_id, encrypted_vet_info, encrypted_user_struct) # rubocop:disable Metrics/MethodLength
```
Comment (kylesoskin) [inline app/sidekiq/central_mail/submit_central_form686c_job.rb:25]: I really like to discourage disabling these rubocop checks. I have never had a situation where I couldn't do some slight refactoring to just pull some stuff out into a function.

Comment (tblackwe) [reply app/sidekiq/central_mail/submit_central_form686c_job.rb]: split the method

**Review by kylesoskin — APPROVED** [review]
Looks good!

**Review by zack — APPROVED** [review]
Everything looks good to me.

**Pull request merged**